gammapy/data/metadata.py

```diff
@@ -0,0 +1,160 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from typing import Optional, Union
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.time import Time
+from pydantic import Field, ValidationError, validator
+from gammapy.utils.fits import earth_location_from_dict
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+from gammapy.utils.time import time_ref_from_dict
+
+__all__ = ["ObservationMetaData"]
+
+
+class ObservationMetaData(MetaData):
+    """Metadata containing information about the Observation.
+
+    Parameters
+    ----------
+    telescope : str, optional
+        the telescope/observatory name
+    instrument : str, optional
+        the specific instrument used
+    observation_mode : str, optional
+        the observation mode
+    location : `~astropy.coordinates.EarthLocation` or str, optional
+        the observatory location
+    deadtime_fraction : float
+        the observation deadtime fraction. Default is 0.
+    time_start : Time, str
+        the observation start time
+    time_stop : Time, str
+        the observation stop time
+    reference_time : Time, str
+        the observation reference time
+    target_name : str
+        the observation target name
+    target_position : SkyCoord
+        the target coordinate
+    creation : `~gammapy.utils.CreatorMetaData`
+        the creation metadata
+    optional : dict
+        additional optional metadata
+    """
+
+    telescope: Optional[str]
+    instrument: Optional[str]
+    observation_mode: Optional[str]
+    location: Optional[Union[str, EarthLocation, None]]
+    deadtime_fraction: float = Field(0.0, ge=0, le=1.0)
+    time_start: Optional[Union[Time, str, None]]
```

Comment:
Optional[...] is a shorthand for Union[None, ...]

So either include None in the Union and remove optional or remove None

Reply:
thanks.